ZenPacks/zenoss/LinuxMonitor/impact.py

```diff
@@ -51,9 +51,14 @@ class FileSystemRelationsProvider(BaseRelationsProvider):
     def getEdges(self):
         filesystem = self._object
         device = filesystem.device()
-        if device.getDeviceClassName() == '/Server/SSH/Linux':
+        logicalvolume = filesystem.getLogicalVolume()
+        if '/Server/SSH/Linux' in device.getDeviceClassName():
```

Comment:
This isn't really related to your change, but there are a couple of preexisting problems with this impact. The first is that device class should never be used to make these kinds of decisions. Users can structure device classes however they want. The second is that filesystems should not impact their device. If anything, the impact should be the other way around.